src/documents/insights/IntegratedSolutions.md

```diff
@@ -6,19 +6,31 @@ eleventyNavigation:
     order: 3
 ---
 
-Too often, design solutions that meet the needs of marginalized users, including users with disabilities, are segregated from the mainstream, resulting in unaffordable solutions and isolation of a particular population. For example, in the past, screen reader-accessible websites were created separately from the main site, often providing the user with a less engaging experience.
+Too often, design solutions that meet the needs of marginalized users, including users with disabilities, are segregated
+from the mainstream, resulting in unaffordable solutions and isolation of a particular population. For example, in the
+past, screen reader-accessible websites were created separately from the main site, often providing the user with a less
+engaging experience.
 
-An inclusive design solution is one that is integrated into mainstream design, making the solution more affordable and more usable for all. For example, an accessible playground that is engaging and fun for all kids, including those with disabilities, allows them to play together rather than segregating some kids into a separate space.
+An inclusive design solution is one that is integrated into mainstream design, making the solution more affordable and
+more usable for all. For example, an accessible playground that is engaging and fun for all kids, including those with
+disabilities, allows them to play together rather than segregating some kids into a separate space.
 
-Creating adaptable and flexible designs is one way to allow for integration. In addition, when diverse participation in the design process is encouraged from the start, the resulting design solutions are much more likely to be fully integrated. By leveraging the curb-cut effect*, a solution created to meet the needs of a particular population becomes more usable for everyone.
+Creating adaptable and flexible designs is one way to allow for integration. In addition, when diverse participation in
+the design process is encouraged from the start, the resulting design solutions are much more likely to be fully
+integrated. By leveraging the curb-cut effect*, a solution created to meet the needs of a particular population becomes
+more usable for everyone.
 
-Integration is often avoided and is left to the end users. This approach can lead to costly modification processes, legal penalties and ultimately misses an opportunity to reach out to many people who might have benefited from that integration.
+Integration is often avoided and is left to the end users. This approach can lead to costly modification processes,
+legal penalties and ultimately misses an opportunity to reach out to many people who might have benefited from that
+integration.
 
 *a ramp from the street to the sidewalk designed for wheelchairs, that also meets the needs of others e.g. cyclists
```

Comment:
Is there a better way we can represent this foot note. It's a bit confusing in the context of this page. I know this isn't specific to this issue, so might be something we want to file separately.

Reply:
I think it would be a good idea to open a higher-level issue (maybe in Fluidic Eleventy?) about how to handle footnotes in Markdown content.

Reply:
https://github.com/fluid-project/eleventy-plugin-fluid/issues/15

Reply:
thanks